src/main/java/com/ranorex/jenkinsranorexplugin/RanorexRunnerBuilder.java

```diff
@@ -98,81 +140,142 @@ public RanorexRunnerBuilder(String rxTestSuiteFilePath, String rxRunConfiguratio
 
     }
 
+    @CheckForNull
     public String getRxTestSuiteFilePath() {
         return this.rxTestSuiteFilePath;
     }
-
+    @CheckForNull
     public String getRxRunConfiguration() {
         return this.rxRunConfiguration;
     }
-
+    @CheckForNull
     public String getRxReportDirectory() {
         return this.rxReportDirectory;
     }
-
+    @CheckForNull
     public String getRxReportFile() {
         return this.rxReportFile;
     }
-
+    @CheckForNull
     public String getRxReportExtension() {
         return this.rxReportExtension;
     }
-
-    public Boolean getrxJUnitReport() {
+    @CheckForNull
+    public Boolean getRxJUnitReport() {
         return this.rxJUnitReport;
     }
-
+    @CheckForNull
     public Boolean getRxZippedReport() {
         return this.rxZippedReport;
     }
-
+    @CheckForNull
     public String getRxZippedReportDirectory() {
         return this.rxZippedReportDirectory;
     }
-
+    @CheckForNull
     public String getRxZippedReportFile() {
         return this.rxZippedReportFile;
     }
-
+    @CheckForNull
     public Boolean getRxTestRail() {
         return this.rxTestRail;
     }
-
+    @CheckForNull
     public String getRxTestRailUser() {
         return this.rxTestRailUser;
     }
-
+    @CheckForNull
     public String getRxTestRailPassword() {
         return this.rxTestRailPassword;
     }
-
+    @CheckForNull
     public String getRxTestRailRID() {
         return this.rxTestRailRID;
     }
-
+    @CheckForNull
     public String getRxTestRailRunName() {
         return this.rxTestRailRunName;
     }
-
+    @CheckForNull
     public String getRxGlobalParameter() {
         return this.rxGlobalParameter;
     }
-
+    @CheckForNull
     public String getCmdLineArgs() {
         return this.cmdLineArgs;
     }
-
-    // public String getRxExecuteableFile()
-    // {
-    // return this.rxExecuteableFile;
-    // }
-
-
+    
+    @DataBoundSetter
+    public void setRxTestSuiteFilePath(@CheckForNull String rxTestSuiteFilePath) {
+    	this.rxTestSuiteFilePath = Util.fixNull(rxTestSuiteFilePath);
+    }
+    @DataBoundSetter
+    public void setRxRunConfiguration(@CheckForNull String rxRunConfiguration) {
+    	this.rxRunConfiguration = Util.fixNull(rxRunConfiguration);
+    }
+    @DataBoundSetter
+    public void setRxReportDirectory(@CheckForNull String rxReportDirectory) {
+    	this.rxReportDirectory = Util.fixNull(rxReportDirectory);
+    }
+    @DataBoundSetter
+    public void setRxReportFile(@CheckForNull String rxReportFile) {
+    	this.rxReportFile = Util.fixNull(rxReportFile);
+    }
+    @DataBoundSetter
+    public void setRxReportExtension(@CheckForNull String rxReportExtension) {
+    	this.rxReportExtension = Util.fixNull(rxReportExtension);
+    }
+    @DataBoundSetter
+    public void setRxJUnitReport(@CheckForNull Boolean rxJUnitReport) {
+    	this.rxJUnitReport = BooleanUtils.isTrue(rxJUnitReport);
```

Comment:
Minor prio: Why are all these BooleanUtils calls needed? It's still storing a Boolean, not a boolean

Reply:
It's a null check - returns false if null.

Reply:
Just asked because it's still stored as a Boolean (object), should be stored as a boolean then IMHO

Reply:
Got it, changed so it's stored as boolean.